application/redis-mysql-helidon/src/test/java/webapp/tier/events/TaskServiceMainTest.java

```diff
@@ -0,0 +1,12 @@
+package webapp.tier.events;
+
+import org.junit.jupiter.api.Test;
+
+class TaskServiceMainTest {
+
+	@Test
+	void testMain() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5923183)